xml/OrganisationRegistrationAgency.xml

```diff
@@ -3123,9 +4452,24 @@ For example: XR-NUTS-UK_E07000105
 
 </narrative>
             </description>
-            <category>HR</category>
+            <category>HR,PT,AT,SK,NL,MT,ES,LT,GR,IE,EE,CY,CZ,FI,DE,LV,GB,SI,BG,SE,FR,HU,PL,LU,BE,IT,RO</category>
```

Comment:
The source XML doesn’t do this – It looks like it just picks the first category. I wonder if we should do that too.

Reply:
Hmm I think it makes sense to capture all of them (rather than just e.g. Croatia). I guess that should be done in a more structured way, to allow multiple categories? But maybe we think about that later, and just stick to this format for now? We could also split a list of categories when displaying on codelists.codeforiati.org

Reply:
Though I wonder if there might be a better delimiter, e.g. `;` ?

Reply:
I changed this to ; as discussed.